modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -66,6 +67,14 @@ public class CatalogUtils {
      */
     public static final String DEFAULT_FILTER = "$..*";
 
+    /**
+     * Default auto adjust scale up timeout of the default zone.
+     */
+    // In case of enabled colocation the start of each node triggers default zone rebalance. In order to eliminate such excessive rebalances
+    // default zone auto adjust scale up timeout is set to 5 seconds. If colocation is disabled tests usually create tables
+    // after all nodes already started meaning that tables are created on stable topology and usually doesn't assume any rebalances at all.
+    public static final int DEFAULT_ZONE_DEFAULT_AUTO_ADJUST_SCALE_UP_TIMEOUT_SECONDS = enabledColocation() ? 5 : 0;
```

Comment:
Just my two cents, in general, such an approach might be a problem. Let's consider a simple test:
```
public class Test {
    @WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "false")
    public vois testPartitionPerTable() {...}

    @WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "true")
    public vois testPartitionPerZone() {...}
}
```

In that case, the value of `DEFAULT_ZONE_DEFAULT_AUTO_ADJUST_SCALE_UP_TIMEOUT_SECONDS` will depend on the test order.

Reply:
Yes it will. And it's somewhat desired. In case of disabled colocation we don't really care about default zone scaleUpTimeout, which is no longer true in case of enabled colocation.

Reply:
I mean that this variable is `static`, therefore it will be initialized only once. Let's assume that the test order is:
 1. `testPartitionPerTable` (the colocation feature is disabled). So, `DEFAULT_ZONE_DEFAULT_AUTO_ADJUST_SCALE_UP_TIMEOUT_SECONDS` will be initialized by `0` (immediate)
 2. `testPartitionPerZone` (enabled colocation). This test will use `0` as well, even though the colocation is enabled.
 
It seems to me that this behaviour is wrong. Am I missing something?

Reply:
I would prefer removing the `static` keyword, but it's up to you.

Reply:
Good catch!
```
    public static int defaultZoneDefaultAutoAdjustScaleUpTimeoutSeconds() {
        return enabledColocation() ? 5 : 0;
    }
```
added instead of DEFAULT_ZONE_DEFAULT_AUTO_ADJUST_SCALE_UP_TIMEOUT_SECONDS